ngraph/core/reference/src/runtime/reference/tile.cpp

```diff
@@ -52,53 +52,72 @@ void runtime::reference::tile(const char* arg,
                               const size_t elem_size,
                               const std::vector<int64_t>& repeats)
 {
-    Shape in_shape_expanded(in_shape);
-    in_shape_expanded.insert(in_shape_expanded.begin(), out_shape.size() - in_shape.size(), 1);
-    size_t block_size = 0;
-    int64_t num_repeats = 0;
-    const int input_rank = in_shape_expanded.size();
-    const int64_t last_dim = in_shape_expanded[input_rank - 1];
-    const std::vector<int64_t> pitches = create_pitches(out_shape);
-    const char* copy = nullptr;
-
-    std::vector<int64_t> indices(in_shape_expanded.size() - 1, 0);
-    size_t axis = indices.size();
-
-    // Copy and repeat data for innermost axis as many times as described in the repeats parameter
-    while (axis <= indices.size())
+    try
```

Comment:
Could you explain why you added this try/catch and changed the Tile code?

Should we add more tests for Tile op?

Reply:
@ilyachur I've observed exception in e2e after merge https://github.com/openvinotoolkit/openvino/pull/2715/commits/0c5e17d42ea196cf4a71f10aa44a8bccac1c1a8f with very limited error message. It is hard to reproduce it locally. This try/catch is only to collect info about error scenario and I'll revert it.
To the number of tests, probably right, we should add more